tools/logger/convert.c

```diff
@@ -136,7 +137,7 @@ static void print_entry_params(FILE *out_fd,
 
 	if (dt < 0 || dt > 1000.0 * 1000.0 * 1000.0)
 		dt = NAN;
-	
+
```

Comment:
this looks like an indendation patch that should be separated

Reply:
Oh it's a white space change, I'll put it to cosmetic commit. My Emacs is configured to remove white space.